src/lib.rs

```diff
@@ -8,8 +8,6 @@
 //! [opencv-rust](https://github.com/kali/opencv-rust/) which generates OpenCV
 //! bindings using a Python script.
 #![deny(missing_docs)]
-#![deny(missing_debug_implementations)]
-#![deny(missing_copy_implementations)]
```

Comment:
I'm not sure about these denies. I removed them when I added global `Cv` struct without any fields, just to create bunch of static methods. Now they can be restored, but i'm not sure if they should.

Reply:
I added these lints after reading [Good Practices for Writing Rust Libraries](https://pascalhertleif.de/artikel/good-practices-for-writing-rust-libraries/).

Users of this library can compose new types and expect debug print works whenever they should.